src/test/java/com/suse/saltstack/netapi/client/SaltStackClientTest.java

```diff
@@ -129,7 +128,7 @@ public void testLoginFailure() throws Exception {
                 .withHeader("Accept", equalTo("application/json"))
                 .withHeader("Content-Type", equalTo("application/json"))
                 .willReturn(aResponse()
-                .withStatus(HttpURLConnection.HTTP_UNAUTHORIZED)));
+                        .withStatus(HttpURLConnection.HTTP_UNAUTHORIZED)));
```

Comment:
Whitespace was added here, and it looks unrelated to your PR - please try to limit your patch to the minimum change needed. Revert this change please.